PKGBUILD

```diff
@@ -6,10 +6,11 @@ url='https://wiki.ros.org/roslz4'
 pkgname='ros-noetic-roslz4'
 pkgver='1.15.4'
 arch=('any')
-pkgrel=1
+pkgrel=2
 license=('BSD')
 
 ros_makedepends=(
+    ros-noetic-cpp-common
```

Comment:
Can you ensure the same style is kept by having the same indentation as the rest of the file?